Server/ReasnAPI/ReasnAPI/Controllers/AuthController.cs

```diff
@@ -0,0 +1,49 @@
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using ReasnAPI.Mappers;
+using ReasnAPI.Models.Authentication;
+using ReasnAPI.Services.Authentication;
+
+namespace ReasnAPI.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
```

Comment:
Maybe do some global route prefix in Program.cs to not write `api` all the time, wdyt?

Reply:
This will be applied in #43.